### PR TITLE
Added context menu to window items

### PR DESCRIPTION
**Pull request by SpaceboyRoss01**

### CODE REVIEW

**Comment by SpaceboyRoss01** [comment]
There's currently a bug where the context menu won't show.

**Comment by SpaceboyRoss01** [comment]
I verified that it worked and I'll be pushing the commit soon.

File: src/locales.js [hunk 1]
```diff
@@ -38,7 +38,13 @@ export const en_EN = {
   LBL_PANELITEM_REMOVE: 'Remove panelitem',
   LBL_PANEL_POSITION: 'Panel Position',
   LBL_SAVE_AND_LOG_OUT: 'Save Session & Log Out',
-  LBL_LOG_OUT: 'Log Out'
+  LBL_LOG_OUT: 'Log Out',
+
```
Comment (andersevenrud) [inline src/locales.js]: I've added all of these strings to the latest `@osjs/client` release. Could you change the usage ?

https://github.com/os-js/osjs-client/commit/09a1170273ca50e72a791151cdc42d53bb50c71c

**Comment by andersevenrud** [comment]
Referring to my last review, it seems like you're pushing your changes one-by-one  -- so you're probably all over this already :nerd_face:

**Comment by SpaceboyRoss01** [comment]
I'm on my Chromebook right now so I'm just updating using the GitHub website.

**Comment by andersevenrud** [comment]
Thanks :blush: :+1:

**Pull request merged**